### ISSUE TITLE
Touch not working on mobile

### ISSUE THREAD

**Issue by KENYONBcom** [Scirra/Construct-3-bugs#4400]
latest version r227.2 when used on my Pixel 3 XL does not behave as it used to in r226.

While viewing the Event Sheet, try swiping from the left to open the properties panel. it starts to show, then springs back immediately. no matter how long you hold down or how slow fast you swipe, it will not appear. What does appear is "Back Navigation" alert. It says Back was pressed and asks to Stay or Leave.

While viewing the Layout, swiping from the right and multiple swipes to show the different panels- Project Layers, Tilemap behaves correctly. The initial swipe from the left to show Properties works, but on another swipe from the left to show the next panel results in the Back behavior. You then have to swipe in from the left again to show the next panel.

TLDR- swiping from left of screen is broken while viewing Event Sheet. Swiping from left of screen while viewing Layout works once, but not with multiple swipes to show next panel.

**Comment by DiegoScirra** [comment]
Fixed in the next beta.
